sky/__init__.py

```diff
@@ -187,6 +191,10 @@ def set_proxy_env_var(proxy_var: str, urllib_var: Optional[str]):
     # core APIs Storage Management
     'storage_ls',
     'storage_delete',
-    'UserTask',
-    'MutatedUserTask',
+    # Admin Policy
+    'UserRequest',
+    'MutatedUserRequest',
+    'AdminPolicy',
+    'NestedConfig',
```

Comment:
For discussion (not feeling strongly): Since this is exposed to users; would sky.Config / sky.GlobalConfig make more sense?  the "nested" part can be inferred from signature & it's an impl detail.

Reply:
Good point! I think calling it `Config` makes sense.